cli/functionary/config.py

```diff
@@ -0,0 +1,59 @@
+from pathlib import Path
+
+import click
+from dotenv import get_key, set_key
+
+
+def save_config_value(key, value):
+    """
+    Save configuration key and value to config file. If key is already present,
+    overwrite the current value.
+
+    Args:
```

Comment:
For consistency with the style we use, the two arguments should be listed separately like:

```
Args:
    key: the configuration parameter to set
    value: the value to set the configuration parameter to
```